client/wp-admin-scripts/wc-pay-payments-promotion/index.tsx

```diff
@@ -0,0 +1,25 @@
+/**
+ * External dependencies
+ */
+import { render, cloneElement } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { WCPaymentsRow } from './wc-payments-row';
+
+const container = document.querySelector(
+	'[data-gateway_id="psuedo_woocommerce_payments"]'
+);
+
+if ( container ) {
+	const sortColumn = container.children[ 0 ].innerHTML;
+	const descriptionColumn = container.children[ 3 ].innerHTML;
```

Comment:
A slightly dirty way to still use the sorting and description from the original payment method row.
React.render overwrites all the elements within the `[data-gateway_id="psuedo_woocommerce_payments"]` row.

This way we can more easily configure the description with the woocommerce.com config, and there won't be a weird jump as the old/new column will look mostly the same.
Only the Install button and the payment icons will be different.